pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -78,6 +79,9 @@ let
                         then "package-db"
                         else "package-conf";
 
+  # the target dir for haddock documentation
+  docdir = docoutput: docoutput + "/share/doc";
```

Comment:
It’s used in `hoogle.nix`, so I’d like to export it; otherwise `hoogle.nix` breaks everytime this changes (around once half a year).